content/id/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Kelola Sertifikat TLS Pada Klaster
+content_template: templates/task
+reviewers:
+- girikuncoro
+---
+
+{{% capture overview %}}
+
+Kubernetes menyediakan API `certificates.k8s.io` yang memungkinkan Kamu membuat sertifikat
+TLS yang ditandatangani oleh Otoritas Sertifikat (CA) yang Kamu kendalikan. CA dan sertifikat ini 
+bisa digunakan oleh _workload_ untuk membangun kepercayaan.
+
+API `certificates.k8s.io` menggunakan protokol yang mirip dengan [konsep ACME](https://github.com/ietf-wg-acme/acme/).
+
+{{< note >}}
+Certificates created using the `certificates.k8s.io` API are signed by a
+dedicated CA. It is possible to configure your cluster to use the cluster root
+CA for this purpose, but you should never rely on this. Do not assume that
+these certificates will validate against the cluster root CA.
+
+Sertifikat yang dibuat menggunakan API `certificates.k8s.io` ditandatangani oleh CA
+khusus. Ini memungkinkan untuk mengkonfigurasi klaster Kamu agar menggunakan CA _root_ klaster untuk tujuan ini,
+namun jangan pernah mengandalkan ini. Jangan berasumsi bahwa sertifikat ini akan melakukan validasi
+dengan CA _root_ klaster  
+{{< /note >}}
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Mempercayai TLS dalam Klaster
+
+Mempercayai CA kustom dari aplikasi yang berjalan sebagai Pod biasanya memerlukan
+beberapa tambahan konfigurasi aplikasi. Kamu harus menambahkan bundel sertifikat CA 
+ke daftar sertifikat CA yang dipercaya klien atau server TLS. 
+Misalnya, Kamu akan melakukan ini dengan konfigurasi TLS golang dengan mengurai rantai sertifikat 
+dan menambahkan sertifikat yang diurai ke `RootCAs` di _struct_
+[`tls.Config`](https://godoc.org/crypto/tls#Config).
+
+Kamu bisa mendistribusikan sertifikat CA sebagai sebuah
+[ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap) yang bisa diakses oleh Pod Kamu.
+
+## Meminta Sertifikat
+
+Bagian berikut mendemonstrasikan cara membuat sertifikat TLS untuk sebuah
+Service kubernetes yang diakses melalui DNS.
+
+{{< note >}}
+Tutorial ini menggunakan CFSSL: PKI dan _toolkit_ TLS dari Cloudflare [klik disini](https://blog.cloudflare.com/introducing-cfssl/) untuk mengetahui lebih jauh.
+{{< /note >}}
+
+## Unduh dan Pasang CFSSL
+
+cfssl yang digunakan pada contoh ini dapat diunduh pada 
```

Comment:
nit: start with uppercase, but since `cfssl` cannot start with uppercase, reorder the sentence
```suggestion
Contoh ini menggunakan cfssl yang dapat diunduh pada 
```